lib/plugins/helper/open_graph.js

```diff
@@ -122,6 +113,20 @@ function openGraphHelper(options = {}) {
     }
   }
 
+  if (author) {
+    result += og('article:author', author);
+  }
+
+  if (keywords) {
+    if (typeof keywords === 'string') keywords = keywords.split(',');
+
+    keywords.map(tag => {
+      return tag.name ? tag.name : tag;
+    }).filter(keyword => !!keyword).forEach(keyword => {
```

Comment:
I might change this to `filter(Boolean)` since it might be a little faster than `filter(x => !!x)`

Reply:
More readable too.